lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -92,38 +92,84 @@ export class PublicClientApplication {
     // #region Redirect Flow
 
     /**
-     * Set the callback functions for the redirect flow to send back the success or error object.
-     * IMPORTANT: Please do not use this function when using the popup APIs, as it will break the response handling
+     * Set the callback functions for the redirect flow to send back the success or error object, and process
+     * any redirect-related data.
+     * IMPORTANT: Please do not use this function when using the popup APIs, as it may break the response handling
      * in the main window.
      * 
      * @param {@link (AuthCallback:type)} authCallback - Callback which contains
      * an AuthError object, containing error data from either the server
      * or the library, depending on the origin of the error, or the AuthResponse object 
      * containing data from the server (returned with a null or non-blocking error).
      */
-    async handleRedirectCallback(authCallback: AuthCallback): Promise<void> {
+    async handleRedirectWithCallback(authCallback: AuthCallback): Promise<void> {
```

Comment:
Seems like an unrelated breaking change? We know there are improvements we want to make to this, might be good to examine holistically (including the name). If we want to change it, I would argue for something more idiomatic to JS callbacks, e.g. `onAuthRedirect` or something.

Reply:
I meant to revert this change, I have a proposal to rename these but I will put it in a separate PR.

Reply:
reverted